### PR TITLE
add style mistake

### PR DESCRIPTION
**Pull request by bryangingechen**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

File: Mathlib/Algebra/Quandle.lean [hunk 1]
```diff
@@ -158,7 +158,7 @@ lemma act_idem (x : S) : (x ◃ x) = x := by rw [← act_one x, ← Shelf.self_d
 
 lemma act_self_act_eq (x y : S) : x ◃ (x ◃ y) = x ◃ y := by
   have h : x ◃ (x ◃ y) = (x ◃ 1) ◃ (x ◃ y) := by rw [act_one]
-  rw [h, ← Shelf.self_distrib, one_act]
+  rw [h, ←Shelf.self_distrib, one_act]
```
[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bors fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bors fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Comment by bryangingechen** [comment]
bot fix style

File: Mathlib/Algebra/Quandle.lean (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

**Review by bryangingechen — COMMENTED** [review]
bot fix style

File: Mathlib/Algebra/Quandle.lean (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

[the remark this answers is not in the document]
Comment (bryangingechen) [reply Mathlib/Algebra/Quandle.lean:161]: bot fix style

**Comment by bryangingechen** [comment]
bot fix style

**Review by bryangingechen — COMMENTED** [review]
bot fix style

**Comment by jcommelin** [comment]
bot fix style

**Comment by jcommelin** [comment]
well, well...

bot fix style

please???

**Comment by jcommelin** [comment]
ding dong

bot fix style

pl0x?

**Pull request closed without merging**